code/_helpers/items.dm

```diff
@@ -0,0 +1,9 @@
+//Prevents robots dropping their modules.
+/proc/dropsafety(var/atom/movable/A)
+	if (istype(A.loc, /mob/living/silicon))
```

Comment:
good to know, but my new function is a generic safety check.

The comment isnt really adequate, aside from preventing borgs dropping their modules (but still allowing the use of the contents of grippers) it also prevents hardsuit modules being dropped.

Its intended to be a central place to hook in any other relevant safety checks, and its run in any situation where stuff is getting dropped into things. Closets, crates, tables, containers, etc.

So, not just for borgs!